core/src/main/java/io/paradoxical/cassieq/discoverable/auth/SignedUrlParameters.java

```diff
@@ -56,7 +61,9 @@ public boolean verify(final VerificationContext context) {
         if (verified) {
             final Instant now = context.getClock().now();
             return startDateTime.map(now::isAfter).orElse(true) &&
-                   endDateTime.map(now::isBefore).orElse(true);
+                   endDateTime.map(now::isBefore).orElse(true) &&
```

Comment:
i think we should encapsulate these predicates, as they are starting to grow. like 

```
notExpired(clock) && queueAllowed()
```